snapshots/client.js

```diff
@@ -11,6 +11,7 @@ window.render = function(source, options, scale) {
 
   var view = scratchblocks.newView(doc, {
     style: options.style,
+    scale: options.scale
```

Comment:
This looks fine!

The `client.js` file is just used for rendering the snapshots, which are a half-baked attempt at snapshot tests. They’re sometimes useful for checking I haven’t broken anything!